chainreg/chainregistry.go

```diff
@@ -411,6 +411,12 @@ func NewPartialChainControl(cfg *Config) (*PartialChainControl, func(), error) {
 				ZMQTxHost:       bitcoindMode.ZMQPubRawTx,
 				ZMQReadDeadline: bitcoindMode.ZMQReadDeadline,
 			},
+			PollingConfig: &chain.PollingConfig{
+				Enable:               bitcoindMode.RPCPolling,
+				BlockPollingInterval: bitcoindMode.BlockPollingInterval,
+				TxPollingInterval:    bitcoindMode.TxPollingInterval,
+				MempoolEvictionAge:   bitcoindMode.MempoolEvictionAge,
+			},
```

Comment:
We'll also want to add some input validation to makes sure users aren't attempting to specify _both_ values.